client/stack-editor/lib/routehandler.coffee

```diff
@@ -1,9 +1,14 @@
 kd = require 'kd'
 lazyrouter = require 'app/lazyrouter'
-
+canCreateStacks = require 'app/util/canCreateStacks'
+isAdmin = require 'app/util/isAdmin'
 
 module.exports = -> lazyrouter.bind 'stackeditor', (type, info, state, path, ctx) ->
 
+  unless canCreateStacks()
+    new kd.NotificationView title: 'You are not allowed to create stacks!'
```

Comment:
This line will throw lint error. `{ title: 'You are not allowed to create stacks!' }`